src/Symfony/Component/ClassLoader/ClassCollectionLoader.php

```diff
@@ -25,12 +25,12 @@ class ClassCollectionLoader
     /**
      * Loads a list of classes and caches them in one big file.
      *
-     * @param array  $classes    An array of classes to load
-     * @param string $cacheDir   A cache directory
-     * @param string $name       The cache name prefix
-     * @param bool   $autoReload Whether to flush the cache when the cache is stale or not
-     * @param bool   $adaptive   Whether to remove already declared classes or not
-     * @param string $extension  File extension of the resulting file
+     * @param array      $classes    An array of classes to load
+     * @param string     $cacheDir   A cache directory
+     * @param string     $name       The cache name prefix
+     * @param bool       $autoReload Whether to flush the cache when the cache is stale or not
+     * @param bool|array $adaptive   Whether to remove already declared classes or not, or array of already declared classes
```

Comment:
There will always be someone doing the not expected things. ;)

The issue is that before they could perform strict comparisons for `true` and `false` being passed. This now would fail when an array got passed.